tests/console/yast2_i.pm

```diff
@@ -25,7 +25,9 @@ sub run {
 
     select_console 'root-console';
 
-    zypper_call "-i inr";    # install recommended drivers bsc#953522
+    # Install recommended drivers bsc#953522 unless when migrating from
+    # versions before 15 (bsc#1127212)
+    zypper_call "-i inr" unless is_sle('<15', get_var('ORIGIN_SYSTEM_VERSION', '15'));
```

Comment:
We have another call on L#33 for jeos. Will it make sense to combine both calls?

Reply:
@ccret Does JeOS need the extra `zypper inr` call, or is the first one enough?

Reply:
@mnowaksuse @rwx788 Calling it one time for JeOS is enough.